### PR TITLE
Add file based readiness probe for worker (for both python and non-python workers)

### PR DESCRIPTION
**Pull request by ro-superblocks**
## Context:
See more context from here https://app.clickup.com/t/8650101/EG-8449

1. Both non-python worker and python worker healthy files are being written.
2. https://github.com/superblocksteam/superblocks/pull/3006/files This PR will enable readiness prob in staging. (This enables it at worker level, so all the fleet type will have this en-abled)

## Tested cases:

Case1:
The following is when both fleet level readinessProbe and worker level readinessProbe are NOT defined (missing this property) The rendered template does not have this field 
Screen Shot 2022-11-16 at 5 59 04 PM

Case2:
The following is when worker level readinessProbe is defined to {}, and it's not defined in fleet level
The rendered template does not have this field 
Screen Shot 2022-11-16 at 6 03 11 PM

Case3: 
The following is when worker level readinessProbe is defined to the correct value (cat /tmp/worker_healthy)
 and it's not defined in fleet level. *We are likely to use this when enabling this in our clould-agent fleet* 
Screen Shot 2022-11-16 at 6 04 29 PM

### CODE REVIEW

**Review by taha-au — CHANGES_REQUESTED** [review]
For now, could you allow the specification of this through values instead? I'd rather start by specifying a values override in our own values yaml file to introduce this given the moving nature of the python worker

File: helm/templates/deployment-worker.yaml [hunk 1]
```diff
@@ -91,6 +91,11 @@ spec:
         - name: tls
           mountPath: /etc/config/tls
         {{- end }}
+        readinessProbe:
```
Comment (bruce-y) [inline helm/templates/deployment-worker.yaml:94]: Do we only want to add a readinessProbe? What about liveness?

Comment (ro-superblocks) [reply helm/templates/deployment-worker.yaml:94]: The current goal is to only handle start up

File: helm/templates/deployment-worker.yaml [hunk 2]
```diff
@@ -91,6 +91,10 @@ spec:
         - name: tls
           mountPath: /etc/config/tls
         {{- end }}
+        {{- if $.Values.worker.readinessProbe }}
+        readinessProbe: {{- toYaml (merge (default (dict) $v.readinessProbe) $.Values.worker.readinessProbe) | nindent 10 }}
+
```
Comment (ro-superblocks) [inline helm/templates/deployment-worker.yaml:96]: remove this from taha

**Pull request closed without merging**